lib/koans/02_strings.ex

```diff
@@ -38,4 +38,8 @@ defmodule Strings do
   koan "Strings can be louder when necessary" do
     assert String.upcase("listen") == ___
   end
+
+  koan "They also support interpolation" do
```

Comment:
Yes, your suggested wording is much better!
I also though about putting it earlier, but after "They can be put together" we have "Or pulled apart when needed". It's good to keep them one after the other.
What about putting the new koan before "They can be put together"?